p2panda-spaces/src/manager.rs

```diff
@@ -251,6 +233,59 @@ where
         Ok(events)
     }
 
+    // @TODO: Make it work without async
+    /// The public key of the local actor.
+    pub async fn id(&self) -> ActorId {
+        let manager = self.inner.read().await;
+        manager.identity.id()
+    }
+
+    /// The local actor id and their long-term key bundle.
+    pub async fn me(&self) -> Result<Member, ManagerError<ID, S, K, M, C, RS>> {
+        let mut manager = self.inner.write().await;
+        manager
+            .identity
+            .me()
+            .await
+            .map_err(ManagerError::IdentityManager)
+    }
+
+    /// Register a member with long-term key bundle material.
+    pub async fn register_member(
+        &self,
+        member: &Member,
+    ) -> Result<(), ManagerError<ID, S, K, M, C, RS>> {
+        let mut manager = self.inner.write().await;
+        manager
+            .identity
+            .register_member(member)
+            .await
+            .map_err(ManagerError::IdentityManager)
+    }
+
+    pub async fn key_bundle_expired(&self) -> bool {
+        let manager = self.inner.read().await;
+        manager.identity.key_bundle_expired().await
+    }
+
+    pub async fn key_bundle(&self) -> Result<M, ManagerError<ID, S, K, M, C, RS>> {
```

Comment:
Maybe name this `key_bundle_message` or similar as this is what the return value is?

Reply:
On a higher-level API we don't really expose key bundles but `Member` to the users, so maybe it makes sense to actually call that method `member_message`?

Reply:
Also, doc-strings will be nice here as well